comictaggerlib/taggerwindow.py

```diff
@@ -1460,7 +1488,8 @@ def populate_combo_boxes(self) -> None:
         self.cbFormat.addItem("Year One")
 
     def remove_auto(self) -> None:
-        self.remove_tags(self.save_data_style)
+        for style in self.save_data_styles:
+            self.remove_tags(style)
```

Comment:
remove_tags needs to take a list of styles instead of doing this. This will cause several popups to happen for every style selected